src/FulfillerMultiSelect.jsx

```diff
@@ -94,7 +94,7 @@ class FulfillerMultiSelect extends React.Component {
                     selectedFulfillerIds
                 },
                 () => {
-                    if (this.state.selectedFulfillerIds.length > 0) {
+                    if (this.state.selectedFulfillerIds.length > 0 || this.props.multi) {
```

Comment:
This is a small consistency improvement for the multi-select mode. Because:
1. In single-select we don't trigger `onChange` when nothing was selected on the initial load
2. In multi-select, we can get `[]` from Customizr as it's a valid selection. In this case, we want to trigger `onChange`, similarly as we do in single-select mode when the value from Customizr was set on the initial load